### ISSUE TITLE
dataview inline query support

### ISSUE THREAD

**Issue by oxideseven** [NicoNekoru/obsidan-advanced-table-xt#47]
Unless I'm just doing it wrong I cannot use dataview inline queries with "Native table post processing"

If I untick that box, the queries work, but the merging doesn't.

Box unticked, the query resolves, but the merge is just a bracket.
![image](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/29870878/62d729f5-2ad8-4641-97c9-e105cf839df8)

Box ticked, the query does not resolve, the merge processes.
![image](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/29870878/64b5edb4-f927-4a00-90ef-3dec4d76793c)

**Comment by beegee115** [comment]
Related problem description, I believe. I'm a new Obsidian user, so I may not have all the details or terminology down. 

My goal was to create a simple table view of the properties of a book note. I have only 3 plugins installed: Book Search, Dataview, and yours (Sheets Extended).

With your plugin activated, I can successfully set up my merged cell table like this (prefixed by an h1 with the book title from the page's properties).

```
# `=this.title`

| col 1 | col 2 | 
| -- | -- |
| image link goes here | title goes here |
| ^    | series or subtitle goes here |
| ^ | date read goes here|
| ^ | review stars here |

```

Here is the output with your plugin turned on: 

<img width="470" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/98f0c9cd-a35c-4d2a-9f35-47af0b3c69d7">

However, as soon as I try to add that title to the appropriate cell of the table, I get `-` instead of the expected `Geekerella`.

```
# `=this.title`

| col 1 | col 2 | 
| -- | -- |
| image link goes here | `=this.title` |
| ^    | series or subtitle goes here |
| ^ | date read goes here|
| ^ | review stars here |
```

<img width="441" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/92799d52-9f0f-4abb-88f9-b699eb150e51">

If I disable **Sheets Extended** the navigate away to another file and back to this one, the dataview query is properly parsed, but (of course) the table cells are no longer merged.

<img width="418" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/7137e376-ae4c-4445-add9-5a63a3b7ca48">

My best guess is that it's a nested plugins issue? I hope my description helps. I believe my end goal is the same as @oxideseven

**Comment by NicoNekoru** [comment]
Is this problem still present?

**Comment by beegee115** [comment]
I can try it again on the weekend. Do you have reason to think the problem has been resolved?

**Comment by NicoNekoru** [comment]
@beegee115 Yes! The way I am trying to operate on the new table editor should preserve dataview's inline rendering although cell merging no longer works. In my local testing it works properly, but I would like to confirm that it is fixed on your end too. Inline queries still do not work properly using code block syntax and I do not plan to add support for that.

**Comment by beegee115** [comment]
@NicoNekoru 

I re-installed (and updated, and enabled) your plugin
<img width="668" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/261d9e5d-36b8-4d47-92d6-6ef2a3e473b6">

Then tried the examples I had copied/pasted into my comments above.

Unfortunately it's not working for me. Cells are not merged, and I saw the title get parsed once for the second table, but then it went away. I restarted Obsidian and it seems to be back now.

```---
title: Geekerella
---

# `=this.title`

| col 1 | col 2 | 
| -- | -- |
| image link goes here | title goes here |
| ^    | series or subtitle goes here |
| ^ | date read goes here|
| ^ | review stars here |


# `=this.title`

| col 1 | col 2 | 
| -- | -- |
| image link goes here | `=this.title` |
| ^    | series or subtitle goes here |
| ^ | date read goes here|
| ^ | review stars here |
```

Results (after restarting Obsidian, the book title appears to remain):

<img width="442" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/92a01f92-1be9-4e4e-8cbe-00ff6d52b0a4">


I have a few other plugins installed now as well as the Minimal theme. Here's the list of what I have installed. I did this test in a folder that doesn't have any automatic templates and isn't set up to "auto" anything.

<img width="151" alt="image" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/3853484/912006fd-7f58-46cb-82f9-722d967839b1">

Best wishes.

**Comment by Nydid** [comment]
Why is inline query support not being considered? I'm using it a lot with the default Obsidian tables, and if this plugin breaks it, that's a dealbreaker.

**Comment by claudermilk** [comment]
New Obsidian user and I've been trying to get this working. After some detective work, I found this plugin to be my issue and found this open issue. Still exists, and since the plugin hasn't been updated for 2 years as of now, I have a feeling it isn't getting fixed.

**Comment by NicoNekoru** [comment]
Should be fixed in 2.0.1
